Samples/auth/Outlook-Event-SSO-NAA/README.md

```diff
@@ -0,0 +1,152 @@
+---
+page_type: sample
+urlFragment: outlook-event-sso-naa
+products:
+  - office
+  - office-outlook
+languages:
+  - javascript
+extensions:
+  contentType: samples
+  technologies:
+    - Add-ins
+  createdDate: "08/20/2024 10:00:00 AM"
+description: "This sample shows how to implement SSO in an event in an Outlook add-in by using nested app authentication."
+---
+
+# Use SSO in events in an Outlook add-in using nested app authentication (preview)
+
+## Summary
+
+This sample shows how to implement SSO in an event in an Outlook add-in. It uses the Microsoft Authentication Library for JavaScript (MSAL.js) and nested app authentication (NAA) to access Microsoft Graph APIs for the signed in user. The sample displays the signed in user's name as a signature in the body of a new email or calendar item.
+
+> [!IMPORTANT]
+> Nested app authentication is currently in preview. To try this feature, you need to join the [Microsoft 365 Insider Program](https://insider.microsoft365.com/join) and choose **Beta Channel**. Don't use NAA in production add-ins. We invite you to try out NAA in test or development environments and welcome feedback on your experience through GitHub (see https://github.com/OfficeDev/office-js/issues).
+
+## Features
+
+- Use MSAL.js NAA to get an access token for the signed in user to call Microsoft Graph APIs.
+- Get an access token through NAA in the OnNewMessageCompose and OnNewAppointmentOrganizer events.
+- Add a signature to an email or calendar invite with the signed in user's name.
```

Comment:
Linda should correct me if I'm wrong, but I believe when signed-in is an adjective, it should be hyphenated.
```suggestion
- Add a signature to an email or calendar invite with the signed-in user's name.
```